src/theme/DocSidebarItem/Category/styles.module.css

```
@@ -0,0 +1,16 @@
+.categoryLink {
+  overflow: hidden;
+}
+
+:global(.menu__link--sublist-caret)::after {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)